geth/account/proxy.go

```diff
@@ -0,0 +1,32 @@
+package account
```

Comment:
What's name `proxy.go` stands for? I'm really confused with what it supposed to mean - `proxy`, `accountNode` and `accountNodeManager` in the same file, and it's hard to understand what's the purpose of this structure. Can you elaborate?

Reply:
Reasonable. I've just reorganized it.